router/debug_router.go

```diff
@@ -24,7 +24,7 @@ func NewDebugRouter(cfg config.Server, a service.Authorizationd) *http.ServeMux
 		dur = time.Second * 3
 	}
 
-	for _, route := range NewDebugRoutes(cfg, a) {
+	for _, route := range NewDebugRoutes(cfg.DebugServer, a) {
```

Comment:
cleaner to split the routes to different function?
```go
if cfg.DebugServer.EnablePolicyDump {
    for _, route := range NewPolicyDumpRoutes(a) {}
}
if cfg.DebugServer.EnableProfiling {
    for _, route := range NewPprofRoutes() {}
}
```

Reply:
I think it is ok to put in one function, maybe only separate the under-layer logic?

Reply:
The logic is very simple, and I will keep the current implementation.